src/pages/workspaces/workspace/DeleteWorkspaceModal.js

```diff
@@ -61,59 +64,112 @@ const DeleteWorkspaceModal = ({ workspace, workspace: { workspace: { name, bucke
-      icon('warning-standard', { size: 24, color: colors.warning() }),
-      span({ style: { marginLeft: '1ch' } }, ['Delete workspace'])
-    ]),
-    onDismiss,
-    okButton: h(ButtonPrimary, {
-      disabled: _.toLower(deleteConfirmation) !== 'delete workspace' || isDeleteDisabledFromResources,
-      onClick: deleteWorkspace,
-      tooltip: Utils.cond(
-        [isDeleteDisabledFromResources && isGoogleWorkspace(workspace), () => 'You must ensure all apps in this workspace are deletable'],
-        [isDeleteDisabledFromResources && isAzureWorkspace(workspace), () => 'This workspace cannot be deleted'],
-        [_.toLower(deleteConfirmation) !== 'delete workspace', () => 'You must type the confirmation message'],
-        () => '')
-    }, 'Delete workspace'),
-    styles: { modal: { background: colors.warning(0.1) } }
-  }, [
-    div(['Are you sure you want to permanently delete the workspace ',
-      span({ style: { fontWeight: 600, wordBreak: 'break-word' } }, name),
-      '?']),
-    getStorageDeletionMessage(),
-    isDeleteDisabledFromResources && div({ style: { marginTop: '1rem' } }, [
-      getResourceDeletionMessage()
-    ]),
-    !isDeleteDisabledFromResources && hasApps() && div({ style: { marginTop: '1rem' } }, [
-      p(['Deleting it will also delete any associated applications:']),
-      getResourceDeletionMessage()
-    ]),
-    collaboratorEmails && collaboratorEmails.length > 0 && div({ style: { marginTop: '1rem' } }, [
-      p(`${pluralize('collaborator', collaboratorEmails.length, true)} will lose access to this workspace.`),
-      div(collaboratorEmails.slice(0, 5).map(
-        email => div({ key: email, style: { overflow: 'hidden', textOverflow: 'ellipsis' } }, [h(Link, { href: `mailto:${email}` }, [email])])
-      )),
-      collaboratorEmails.length > 5 && (
-        div(`and ${collaboratorEmails.length - 5} more`)
-      )
-    ]),
-    !isDeleteDisabledFromResources && b({ style: { display: 'block', marginTop: '1rem' } }, 'This cannot be undone.'),
-    !isDeleteDisabledFromResources && div({ style: { display: 'flex', flexDirection: 'column', marginTop: '1rem' } }, [
-      label({ htmlFor: 'delete-workspace-confirmation', style: { marginBottom: '0.25rem' } }, ['Please type \'Delete Workspace\' to continue:']),
-      h(TextInput, {
-        id: 'delete-workspace-confirmation',
-        placeholder: 'Delete Workspace',
-        value: deleteConfirmation,
-        onChange: setDeleteConfirmation
-      })
-    ]),
-    (deleting || loading) && spinnerOverlay
-  ])
+  const renderAzureCleanupModal = () => {
+    return h(Modal, {
+      title: span({ style: { display: 'flex', alignItems: 'center' } }, [
+        icon('warning-standard', { size: 24, color: colors.warning() }),
+        span({ style: { marginLeft: '1ch' } }, ['Delete workspace resources'])
+      ]),
+      onDismiss,
+      okButton: h(ButtonPrimary, {
+        disabled: isDeleteDisabledFromResources,
```

Comment:
If the button is disabled, should have a tooltip that explains why.

Reply:
Added here https://github.com/DataBiosphere/terra-ui/pull/3800/commits/4d41938531b5aa324615d06e9d9c3b3a0b0ff4eb#diff-4ac38477d4952cecb992aa5760563f223b8f653b9fd2a2a030782e825284bfb3R83